web/public/mgmt/config.yml

```diff
@@ -4273,8 +4288,6 @@ collections:
               - { label: Link One, name: linkOne, widget: string }
               - { label: Link Two Text, name: linkTwoText, widget: string }
               - { label: Link Two, name: linkTwo, widget: string }
-              - { label: Link Three Text, name: linkThreeText, widget: string }
-              - { label: Link Three, name: linkThree, widget: string }
               - { label: Made With Text, name: madeWithText, widget: string }
```

Comment:
This was ripped out a while ago (can you global search for the names), just not removed from config.yml